### PR TITLE
Removed unnecessary parameters on call_command in some tests

### PR DESCRIPTION
**Pull request by MyungSeKyo**

### CODE REVIEW

File: tests/admin_scripts/tests.py [hunk 1]
```diff
@@ -1353,11 +1353,9 @@ class ManageTestserver(SimpleTestCase):
 
     @mock.patch.object(TestserverCommand, 'handle', return_value='')
     def test_testserver_handle_params(self, mock_handle):
-        out = StringIO()
-        call_command('testserver', 'blah.json', stdout=out)
+        call_command('testserver', 'blah.json')
         mock_handle.assert_called_with(
-            'blah.json',
-            stdout=out, settings=None, pythonpath=None, verbosity=1,
+            'blah.json', settings=None, pythonpath=None, verbosity=1,
```
Comment (felixxm) [inline tests/admin_scripts/tests.py:1358]: This is not unnecessary, we check if command is called with specific params.

File: tests/user_commands/tests.py [hunk 2]
```diff
@@ -119,7 +119,7 @@ def test_call_command_option_parsing_non_string_arg(self):
         It should be possible to pass non-string arguments to call_command.
         """
         out = StringIO()
-        management.call_command('dance', 1, verbosity=0, stdout=out)
+        management.call_command('dance', 1, stdout=out)
```
Comment (felixxm) [inline tests/user_commands/tests.py:122]: Only this parameter is unnecessary, rest of them is necessary to hide std outputs.

**Comment by MyungSeKyo** [comment]
@felixxm 
I am so sorry. I didn't know that.
I'v only left the necessary modifications.

File: tests/user_commands/tests.py [hunk 3]
```diff
@@ -171,10 +171,10 @@ def test_check_migrations(self):
         self.assertIs(requires_migrations_checks, False)
         try:
             with mock.patch.object(BaseCommand, 'check_migrations') as check_migrations:
-                management.call_command('dance', verbosity=0)
+                management.call_command('dance')
                 self.assertFalse(check_migrations.called)
                 dance.Command.requires_migrations_checks = True
-                management.call_command('dance', verbosity=0)
```
Comment (felixxm) [inline tests/user_commands/tests.py:177]: Please revert this change, `verbosity` is necessary.

File: tests/user_commands/tests.py [hunk 4]
```diff
@@ -171,10 +171,10 @@ def test_check_migrations(self):
         self.assertIs(requires_migrations_checks, False)
         try:
             with mock.patch.object(BaseCommand, 'check_migrations') as check_migrations:
-                management.call_command('dance', verbosity=0)
```
Comment (felixxm) [inline tests/user_commands/tests.py:174]: Please revert this change, `verbosity` is necessary.

File: tests/user_commands/tests.py [hunk 5]
```diff
@@ -159,9 +159,9 @@ def patched_check(self_, **kwargs):
         saved_check = BaseCommand.check
         BaseCommand.check = patched_check
         try:
-            management.call_command("dance", verbosity=0)
+            management.call_command("dance")
```
Comment (felixxm) [inline tests/user_commands/tests.py:162]: Please revert this change, `verbosity` is necessary.

File: tests/user_commands/tests.py [hunk 6]
```diff
@@ -159,9 +159,9 @@ def patched_check(self_, **kwargs):
...
             self.assertEqual(self.counter, 0)
-            management.call_command("dance", verbosity=0, skip_checks=False)
+            management.call_command("dance", skip_checks=False)
```
Comment (felixxm) [inline tests/user_commands/tests.py:164]: Please revert this change, `verbosity` is necessary.

File: tests/user_commands/tests.py [hunk 7]
```diff
@@ -44,7 +44,7 @@ def test_command_style(self):
 
     def test_language_preserved(self):
         with translation.override('fr'):
-            management.call_command('dance', verbosity=0)
+            management.call_command('dance')
```
Comment (felixxm) [inline tests/user_commands/tests.py:47]: Please revert this change, `verbosity` is necessary.

File: tests/user_commands/tests.py (hunk 2 above)
Comment (felixxm) [reply tests/user_commands/tests.py:122]: Here we display a message even with `verbosity=0`, so I think it's worth to leave it.
